coins.json

```
@@ -1593,5 +1593,30 @@
       "clientPublic": "https://seed.thorchain.info",
       "clientDocs": ""
     }
+  },
+  {
+    "id": "cryptoorg",
+    "name": "Crypto_org",
+    "coinId": 394,
+    "symbol": "CRO",
+    "decimals": 8,
+    "blockchain": "Cosmos",
+    "derivationPath": "m/44'/394'/0'/0/1",
```

hi @Galadrin , i am one of developer of Crypto.com, and we used wallet-core for a long time, also we supported CRO before mainnet release for testing already but with `"derivationPath": "m/44'/394'/0'/0/0"`. could you update it to same as that we using now?
```suggestion
    "derivationPath": "m/44'/394'/0'/0/0",
```